core/payment-driver/src/gnt/sender.rs

```diff
@@ -106,6 +108,7 @@ pub struct TransactionSender {
 impl TransactionSender {
     pub fn new(ethereum_client: Arc<EthereumClient>, db: DbExecutor) -> Addr<Self> {
         let me = TransactionSender {
+            active_accounts: Default::default(),
```

Comment:
I believe accounts can be already unlocked when the service is starting. We need to send `List` message to the identity API and check for unlocked accounts during startup.